client/zeth/core/contracts.py

```diff
@@ -220,14 +220,13 @@ def mix_parameters_as_contract_arguments(
     Convert MixParameters to a list of eth ABI objects which can be passed to
     the contract's mix method.
     """
-    proof_params: List[Any] = zksnark.proof_to_contract_parameters(
-        mix_parameters.extended_proof)
-    proof_params.extend([
+    proof_params: List[Any] = [
```

Comment:
What about renaming this one as `mix_params`?

Reply:
renaed to `mix_params`